apps/vscode-extension/package.json

```diff
@@ -123,10 +132,6 @@
       {
         "command": "otter.menu.generate.store",
         "title": "Generate Store"
-      },
-      {
-        "command": "otter.menu.extract.styling.variable",
```

Comment:
should not be removed

Reply:
We found a way to specify Otter: Only in the Command Panel by adding a category so we do not need another command to have different names :)